aws/src/integration/java/org/apache/iceberg/aws/TestAssumeRoleAwsClientFactory.java

```diff
@@ -117,7 +65,7 @@ public void testAssumeRoleGlueCatalog() throws Exception {
       Assert.assertEquals(AccessDeniedException.class, e.getClass());
     }
 
-    Namespace namespace = Namespace.of("allowed_" + UUID.randomUUID().toString().replace("-", ""));
+    Namespace namespace = Namespace.of("iceberg_aws_ci_" + UUID.randomUUID().toString().replace("-", ""));
```

Comment:
Nit: iceberg_aws_ci_ maybe could be a static prefix? Not blocking though considering it was already like that